### ISSUE TITLE
Disabled buttons for the keypad?

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/area-model-common#138]
I vaguely remember noting that we could disable keys that wouldn't do anything when pressed, and this was noted in the review comments:

```
     * REVIEW: If I'm understanding the intent here, it seems it would be a better user experience to gray out keys
     * REVIEW: that cannot lead to valid sequences.
```

@amanda-phet and @ariel-phet, can you comment? Did we discuss this? (@samreid, thoughts?)

**Comment by amanda-phet** [comment]
My recommendation is to keep the keypad behavior as it is.

**Comment by jbphet** [comment]
@amanda-phet - I'm going through and assigning some keypad issues based on an effort to try to finalize the implementation of the reusable, versatile UI component (see https://github.com/phetsims/scenery-phet/issues/283 for the long and complex history of this issue).  Should this issue be included as one that is reassigned for followup?

**Comment by amanda-phet** [comment]
@jbphet yes! I think this would be good to followup with, especially given @ariel-phet 's experience with the keypad.

**Comment by amanda-phet** [comment]
Seems this should've been assigned back to @jbphet . If this work is deferred then please feel free to unassigned.

**Comment by jbphet** [comment]
I'm not entirely clear on what needs to be done here.  Above I'm seeing some specific discussions about the behavior of the keypad in the area model sims, but I'm not seeing anything that says, for instance, "What we need is the ability to specify which buttons on the keypad are enabled and which ones are disabled."  Is that what is being requested?

**Comment by amanda-phet** [comment]
This sim is in RC right now so let's defer. 

I believe a discussion is in order, but the request was to make some buttons disabled if they were not valid.
